src/controllers/playback/video/index.js

```diff
@@ -1775,6 +1775,15 @@ export default function (view) {
         }
     });
 
+    nowPlayingPositionSlider.addEventListener('keydown', function (e) {
+        if (e.defaultPrevented) return;
```

Comment:
an alternative here would be to call `stopImmediatePropagation()` in emby-slider.js, but I'm guessing we'd prefer not to updated a core component's source if possible?

Reply:
Technically, keyboard dragging should probably do `stopImmediatePropagation()`, since it is an internal behaviour that overrides the standard one. But I would use your approach because it is currently safer. Moreover, one day emby-slider will be replaced by another one from the MUI.

Reply:
Alright, sounds good!